src/wasm-emscripten.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef wasm_emscripten_h
+#define wasm_emscripten_h
+
+#include "wasm-traversal.h"
+
+namespace wasm {
+
+class Module;
+
+void generateMemoryGrowthFunction(Module&);
```

Comment:
It would be nice to put these methods in their own subnamespace or class, so `wasm` doesn't get cluttered. see `ast_utils.h` for some examples, would be nice to be consistent.